src/modules/roc_pipeline/target_stdio/roc_pipeline/parse_address.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2015 Roc authors
```

Comment:
* This code is not stdio-specific, so it should not be under `target_stdio`.
* This code is specific to packet::Address and is not specific to roc_pipeline, so it should be in `roc_packet`.
* s/2015/2019/ here and in `parse_address.h`.

Reply:
Done